src/Buttons/IconButton.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { StyleSheet, TouchableHighlight, View } from 'react-native'
+
+type Props = {
+  icon: any,
+  style: StyleSheet.Styles,
+  onPress: Function
+}
+type State = {
+  pressed: boolean
+}
+
+class IconButton extends Component<Props, State> {
+  static propsTypes = {
+    icon: PropTypes.string.isRequired,
```

Comment:
These proptypes don't match the Flow types. Also, aren't PropTypes outdated technology?

Reply:
Yeah I didn't realize they were in there.